### ISSUE TITLE
Base.show() for Polyhedron instance issue

### ISSUE THREAD

**Issue by Boomer91** [JuliaPolyhedra/Polyhedra.jl#250]
Creating and displaying a polyhedron instance seems to crash the REPL.

MWE:

```julia
using Polyhedra
import GLPK
lib = DefaultLibrary{Float64}(GLPK.Optimizer)

P1 = polyhedron(vrep([1 2]), lib);

display(P1)
```

**Comment by blegat** [comment]
I guess this is a duplicate of https://github.com/JuliaPolyhedra/Polyhedra.jl/issues/237, https://github.com/JuliaPolyhedra/Polyhedra.jl/issues/230 and https://github.com/JuliaPolyhedra/Polyhedra.jl/issues/216.
Should be fixed in Julia v1.7, see https://github.com/JuliaLang/julia/issues/39349

**Comment by blegat** [comment]
See https://github.com/JuliaPolyhedra/Polyhedra.jl/issues/216#issuecomment-1066128472
